packages/functional-tests/src/app.ts

```diff
@@ -116,44 +116,42 @@ export default class App {
     private async displayFunctionalTestChoices(rootActor: MRESDK.ActorLike): Promise<string> {
         return new Promise<string>((resolve) => {
             let y = 0;
-            for (const key in this.testFactories) {
-                if (this.testFactories.hasOwnProperty(key)) {

```

Comment:
>this.testFactories.hasOwnProperty(key) [](start = 20, length = 38)

I thought this was required, TS-lint complaining if missing?

Reply:
It's required with for..in, but not for..of.